Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN set -eux \
 	&& if [ "${TG_VERSION}" = "latest" ]; then \
 		VERSION="$( git describe --abbrev=0 --tags )"; \
 	else \
-		VERSION="$( git tag | grep -E "v${TG_VERSION}\.[.0-9]+" | sort -u | tail -1 )" ;\
+		VERSION="$( git tag | grep -E "v${TG_VERSION}\.[.0-9]+" | sort -Vu | tail -1 )" ;\
```

Comment:
Thanks for this catch! :+1: 

I will have to give this some more throrough investigation as it should actually have been caught by CI via this check:

https://github.com/cytopia/docker-terragrunt/blob/master/Makefile#L71

Reply:
will you fix that in your own PR then?